Makefile

```diff
@@ -5,6 +5,9 @@ DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg http
 
 PIP?=pip
 
+VENV_NAME:=.venv$(VENV_PY_VERSION)
```

Comment:
ok. I did not realize that I didn't see the := and though that lazy eval would be used,
 
Yesterday. I did some experiment to replace make virtualenv by pyenv.
Don't you think it could be a good alternative to custom makefile?